src/test/java/de/mediathekview/mserver/testhelper/AssertFilm.java

```diff
@@ -52,4 +58,73 @@ public static void assertEquals(final Film aActualFilm,
         assertThat(aActualFilm.getSubtitles().toArray(new URL[0])[0].toString(), equalTo(aExpectedSubtitle));
       }      
   }
+  
+  public static void assertEquals(final Film aActualFilm,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 19 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)